odl/test/solvers/functional/default_functionals_test.py

```diff
@@ -505,6 +505,23 @@ def test_moreau_envelope_l2_sq(space, sigma):
     assert all_almost_equal(smoothed_l2_sq.gradient(x),
                             x * 2 / (1 + 2 * sigma))
 
+def test_weighted_proximal():
+    """Test for the weighted proximal of a SeparableSum functional."""
+
+    space = odl.space.rn(2)
+
+    l1 = odl.solvers.L1Norm(space)
+    l2 = odl.solvers.L2Norm(space)
+
+    x = space.one()
```

Comment:
Weird ordering, try to do it "in sequence" so to say:

* define space
* define operators/functionals
* define values
* call operator on values